Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/ui/onboarding/validationrules/info/ValidationRulesInfoFragment.kt

```diff
@@ -0,0 +1,30 @@
+package de.rki.coronawarnapp.covidcertificate.ui.onboarding.validationrules.info
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.ValidationRulesInfoFragmentBinding
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBinding
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import javax.inject.Inject
+
+class ValidationRulesInfoFragment : Fragment(R.layout.validation_rules_info_fragment), AutoInject {
```

Comment:
I think this screen should be `DateTimeInfoFragment` right ? we should have an empty class already for that